spark/src/main/scala/geotrellis/spark/buffer/BufferTiles.scala

```diff
@@ -27,6 +29,7 @@ import org.apache.spark.storage.StorageLevel
 
 import scala.reflect.ClassTag
 import scala.collection.mutable.ArrayBuffer
+import scala.util.Try
```

Comment:
Glad that didn't stick around :)  `Try` and `prototype` are dangling imports now.